src/calculateRentalCost.js

```diff
@@ -5,6 +5,15 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const cost = days * 40;
```

Comment:
Consider using a constant for the daily cost value to avoid hardcoding. For example, 'const DAILY_COST = 40;'.

Reply:
Still not fixed